views/tx.tmpl

```diff
@@ -9,7 +9,7 @@
     <div class="row">
         <div class="col-md-12">
             <h4>Transaction</h4>
-            <p>{{.TxID}}   <span style="font-size: 12px;"><a href="/api/tx/{{.TxID}}?indent=true">view raw</a></span> <span style="background-color: #7FFFD4; border: 1px solid black"><strong>{{$.Type}}</strong></span> </p>
+            <p>{{.TxID}}   <span style="font-size: 12px;"><a href="/api/tx/{{.TxID}}?indent=true">view raw</a></span> <span id ="type-tag" style="background-color: #7FFFD4; border: 1px solid black"><strong>{{.Type}}</strong></span> </p>
```

Comment:
we still need to make this box a bit prettier.  There needs more padding or border or whatever on the inside between the text and the border.  Maybe add ` padding: 2px 4px; border-radius: 3px; box-shadow: 1px 1px 1px grey;` to style.